README.md

```diff
@@ -43,7 +43,7 @@ Will run the following build scripts in order:
 Alternatively run those scripts individually to build the API data in `dist/api/` and HTML content in `dist/` and cache the API responses in `.cache`.
 
 ```
-yarn deploy
+yarn deploy -r talk.sciencegossip.org
```

Comment:
is the `-r` needed here?

Reply:
Yeah. The alternative would be to specify it in the deploy script in `package.json`, but then that would limit the args that you can pass to the batch uploader so 🤷‍♀️ .